mldsa/symmetric.h

```diff
@@ -39,15 +27,19 @@ __contract__(
 #define STREAM128_BLOCKBYTES SHAKE128_RATE
 #define STREAM256_BLOCKBYTES SHAKE256_RATE
 
-#define stream128_init(STATE, SEED, NONCE) \
-  mldsa_shake128_stream_init(STATE, SEED, NONCE)
-#define stream128_squeezeblocks(OUT, OUTBLOCKS, STATE) \
-  shake128_squeezeblocks(OUT, OUTBLOCKS, STATE)
 #define stream256_init(STATE, SEED, NONCE) \
   mldsa_shake256_stream_init(STATE, SEED, NONCE)
 #define stream256_squeezeblocks(OUT, OUTBLOCKS, STATE) \
   shake256_squeezeblocks(OUT, OUTBLOCKS, STATE)
 
+#define mld_xof_ctx keccak_state
+#define mld_xof_init(CTX) shake128_init(CTX)
+#define mld_xof_absorb(CTX, IN, INBYTES) \
+  shake128_absorb(CTX, IN, INBYTES);     \
```

Comment:
Add `do { ... } while(0)` so there's no risk of breaking this in `for` or `if` block without `{ ... }`.

Reply:
Yes, good point. Later we may want to refactor this to an absorb_once though.